### PR TITLE
[geometry-1] Re-add scaleNonUniform for compat reasons to SVGMatrix. …

### PR DESCRIPTION
**Pull request by dirkschulze**
…#214

### CODE REVIEW

File: geometry/Overview.bs [hunk 1]
```diff
@@ -714,11 +714,13 @@ interface DOMMatrixReadOnly {
                                     optional unrestricted double ty = 0,
                                     optional unrestricted double tz = 0);
     [NewObject] DOMMatrix scale(optional unrestricted double scaleX = 1,
-                                optional unrestricted double scaleY,
+                                optional unrestricted double scaleY = 1,
```
Comment (zcorpan) [inline geometry/Overview.bs]: Can you make this a separate issue/pr? I don't recall if this was intentional but that should be considered properly.

File: geometry/Overview.bs [hunk 2]
```diff
@@ -1329,6 +1331,21 @@ export>is 2D</dfn>. This flag indicates that:
 
   <p>The current matrix is not modified.
 
+ <dt><dfn>scaleNonUniform(<var>scaleX</var>, <var>scaleY</var>)</dfn>
+ <dd>
+  <p class=note>Supported for legacy reasons to be compatible with {{SVGMatrix}} as defined in SVG 1.1 [[SVG11]]. Authors should use {{scale()}} instead.
```
Comment (zcorpan) [inline geometry/Overview.bs]: Don't use should in a note

**Comment by dirkschulze** [comment]
@zcorpan Reverted default init of [`scale()`](https://drafts.fxtf.org/geometry/#dom-dommatrixreadonly-scale) for `scaleY`. [`scaleSelf()`](https://drafts.fxtf.org/geometry/#dom-dommatrix-scaleself) indeed specifies that a missing `scaleY` defaults to `scaleX`. Shouldn't this be specified for [`scale()`](https://drafts.fxtf.org/geometry/#dom-dommatrixreadonly-scale) explicitly as well?

File: geometry/Overview.bs [hunk 3]
```diff
@@ -1333,7 +1333,7 @@ export>is 2D</dfn>. This flag indicates that:
 
  <dt><dfn>scaleNonUniform(<var>scaleX</var>, <var>scaleY</var>)</dfn>
  <dd>
-  <p class=note>Supported for legacy reasons to be compatible with {{SVGMatrix}} as defined in SVG 1.1 [[SVG11]]. Authors should use {{scale()}} instead.
+  <p class=note>Supported for legacy reasons to be compatible with {{SVGMatrix}} as defined in SVG 1.1 [[SVG11]]. It is recommended that authors use {{scale()}} instead.
```
Comment (zcorpan) [inline geometry/Overview.bs]: Recommended is still a normative keyword 🙂 say encouraged

**Comment by zcorpan** [comment]
The scaleY change is likely correct, I just couldn't say offhand

**Pull request merged**